packages/rocket-webhook-core/src/webhook-dispatcher.ts

```diff
@@ -20,26 +21,29 @@ export async function dispatch(
     const handlerClass = webhookParamsEvent.handlerClass
     const webhookEvent = toWebhookEvent(webhookParamsEvent, request)
     const register = new Register(requestId, {})
-    const result = await handlerClass.handle(webhookEvent, register)
+    const result: WebhookHandlerReturnType | void = await handlerClass.handle(webhookEvent, register)
     await RegisterHandler.handle(config, register)
+    if (result) {
+      return config.provider.api.requestSucceeded(result.body, result.headers)
+    }
     return config.provider.api.requestSucceeded(result)
```

Comment:
I guess in this case we are calling `requestSucceeded` always with the first parameter set to `undefined`, correct? 

If so, could you please change this to `config.provider.api.requestSucceeded()` (if possible by the parameter type)?

This way is clearer we are sending an empty response (I got a bit confused)

Reply:
Sure, good point